src/ts/src/common/types/ui.ts

```diff
@@ -0,0 +1,16 @@
+import { FC } from "react";
```

Comment:
changed the file from `ui.d.ts` to `ui.ts`, so it's more explicit where the type is coming from